docs/apache-airflow-providers-postgres/connections/postgres.rst

```diff
@@ -29,7 +29,7 @@ Host (required)
     The host to connect to.
 
 Schema (optional)
-    Specify the schema name to be used in the database.
+    Specify the database name.
```

Comment:
Note that the field in the connection will still be `schema`. This is due to how we build the connection string for all databases. But I updated the docs to make sure that it should contain the database name.

Reply:
Should we add a reason in the docs as well on why we are using `schema` field to set `database`?

Reply:
Thanks @kaxil I have updated the docs. PTAL :)